### ISSUE TITLE
Update sbt version to 1.3.6

### ISSUE THREAD

**Issue by gavinbaumanis** [akka/akka-quickstart-scala.g8#42]
The current readme shows 
```
sbt -Dsbt.version=0.13.17 new akka/akka-quickstart-scala.g8
```

and the build.properties, has;
```
sbt.version=1.1.6
```

These two versions are not the same and a lagging behind the latest release, 1.3.6

**Comment by gavinbaumanis** [comment]
I have created a PR for this.

**Comment by raboof** [comment]
fixed with #43
